utils/import_submodules.py

```diff
@@ -0,0 +1,53 @@
+import sys
+import pkgutil
+import importlib
+
+
+def import_submodules(package, skip_modules=None, recursive=True):
+    """
+    Import all submodules of a module, recursively, including subpackages
+
+    Original form is from StackOverflow:
+        https://stackoverflow.com/a/25562415
+
+    Parameters
+    ----------
+
+    package : str or module name
+        name of the package or the imported package to import the
+        submodules from.
+    skip_modules : coma separated str or None
+        Coma separated string of module names to be skipped.
```

Comment:
typo: comma

Reply:
I'm happy that I need to rebase it anyway ;)